### PR TITLE
Add workflow for issue label comments

### PR DESCRIPTION
**Pull request by hbalenda**
Includes automated comments for the status issue labels.

### CODE REVIEW

File: .github/workflows/LABEL_COMMENTS.yml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: Add "status:triage needed" comment
+        uses: peter-evans/create-or-update-comment@5f728c3dae25f329afbe34ee4d08eef25569d79f
+        with:
+          issue-number: ${{ github.event.issue.number }}
+          body: |
+            Your bug is important to us, but we can't get to it quite yet. Stay tuned. We will confirm when your bug has been selected for development.
```
Comment (Erin-Cecele) [inline .github/workflows/LABEL_COMMENTS.yml:79]: This is the message for "backlog." The message for "triage needed" is: “Thank you for your contribution. Your support of Quire is greatly appreciated. We are reviewing your suspected bug and will be in touch soon.”

File: .github/workflows/LABEL_COMMENTS.yml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: Add "status:backlog" comment
+        uses: peter-evans/create-or-update-comment@5f728c3dae25f329afbe34ee4d08eef25569d79f
+        with:
+          issue-number: ${{ github.event.issue.number }}
+          body: |
+            Thank you for your contribution. Your support of Quire is greatly appreciated. We are reviewing your suspected bug and will be in touch soon.
```
Comment (Erin-Cecele) [inline .github/workflows/LABEL_COMMENTS.yml:19]: This is the message for "triage needed." The message for "backlog" is: "Your bug is important to us, but we can’t get to it quite yet. Stay tuned. We will confirm when your bug has been selected for development.” I also think we can remove the "stay tuned." It seems overly wordy.

**Review by Erin-Cecele — CHANGES_REQUESTED** [review]
Hi Hannah, this looks good! Although I think the messages for "backlog" and "triage needed" were accidentally switched.

**Comment by hbalenda** [comment]
Oop, fixed! Thanks @Erin-Cecele

**Review by Erin-Cecele — APPROVED** [review]

**Pull request merged**